ember.md

```diff
@@ -183,3 +185,25 @@ CSS is permitted (and encouraged) in apps and addons under certain circumstances
 </div>
 
 ```
+<a name="testing"></a>
+## Testing
+
+<a name="testing--test-scripts"></a>
+### 5.1 Test Scripts
+
+> Why? To allow us to establish and hold to a standard of code coverage in all of our apps with meaningful test writing and the ability to gate deployments when those standards are not met.
+
+We use three devDependencies to compile our test scripts.  
```

Comment:
I think it would make more sense to say that we use those dependencies to _run_ our testing scripts